spec/services/monthly_district_data_service_spec.rb

```diff
@@ -3,11 +3,26 @@
 RSpec.describe MonthlyDistrictDataService do
   let(:organization) { FactoryBot.create(:organization) }
   let(:facility_group) { create(:facility_group, organization: organization) }
-  let(:facility) { create(:facility, facility_group: facility_group) }
```

Comment:
These specs were pretty weak so I tried to improve them. It's difficult to get the data set up correctly to test everything and didn't seem worth the time, but this is still an improvement I think.